include/ggml/ggml.h

```diff
@@ -524,7 +538,15 @@ extern "C" {
     // next prime after GGML_MAX_NODES
     // #define GGML_GRAPH_HASHTABLE_SIZE 4099
     // next prime after GGML_MAX_NODES * 2 (nodes + leafs)
-    #define GGML_GRAPH_HASHTABLE_SIZE 8273
+    // #define GGML_GRAPH_HASHTABLE_SIZE 8273
+    // #define GGML_GRAPH_HASHTABLE_SIZE 16411
+    #define GGML_GRAPH_HASHTABLE_SIZE 32771
```

Comment:
There is a chance that this and the increase to `GGML_MAX_NODES` will break the examples that allocate the graph on the stack.

Reply:
That's true. I think we will migrate things as we go, or alternatively - migrate everything after #547 is merged

Reply:
Yeah.. I used heap allocation because of this.

We could rewrite the examples to use heap allocated graphs with `ggml_new_graph`.

Maybe it would be more convenient to have a way to dynamically grow the graph beyond some stack allocatable initial capacity. But then all code that iterates over or adds/deletes nodes need to be changed. E.g. by replacing with calls to new API functions for accessing graph nodes. Sounds too bloated.

Or change the build process to add libraries with custom compile definitions for the sizes and let finetune and train-text-from-scratch link against those.

Reply:
The `gpt-2` example has already been updated to allocate on the heap. We just need to apply the same treatment to the rest of the examples